### PR TITLE
[FIX] Visitor and agent info fixed in livechat room

### LINKED ISSUE #14606: "Visitor Info" is empty when clicking the name user in live chat room
### Description:

"Visitor Info" is empty when clicking the name user in live chat room

### Steps to reproduce:

1. Go to a live chat room with someone as agent
2. Click on the visitor's name
3. Visitor info is empty ( you can show it up by the top bar "info" icon though)

### Expected behavior:

Same as clicking the top bar 'info' icon, the visitor info shows up

### Actual behavior:

Gives empty visitor info when clicking on the user in the live chat window

### Server Setup Information:

- Version of Rocket.Chat Server: 1.0.3
- Operating System: ubuntu
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### PR DESCRIPTION
**Pull request by knrt10**
Closes #14606. 

cc @tassoevan @ggazzo would you please review

### CODE REVIEW

File: app/models/server/models/LivechatVisitors.js [hunk 1]
```diff
@@ -57,6 +57,12 @@ export class LivechatVisitors extends Base {
 		return this.find(query);
 	}
 
+	findByUsername(username, options) {
+		const query = { username };
+
+		return this.find(query, options);
```
Comment (renatobecker) [inline app/models/server/models/LivechatVisitors.js]: Why not returning just one Livechat visitor?
```suggestion
		return this.findOne(query, options);
```

Comment (knrt10) [reply app/models/server/models/LivechatVisitors.js]: Yes, Let me update this

Comment (knrt10) [reply app/models/server/models/LivechatVisitors.js]: @renatobecker if we use `findOne` here, the code will break. Then `getFullUserData(...).observeChanges` won't be a function.

**Comment by ashwaniYDV** [comment]
https://github.com/RocketChat/Rocket.Chat/pull/16797

**Comment by engelgabriel** [comment]
fixed by #16797

**Pull request closed without merging**